examples/controlnet/README.md

```diff
@@ -0,0 +1,257 @@
+# ControlNet training example
```

Comment:
Nice doc page!

Reply:
Could you maybe add a W&B link here?

Reply:
https://github.com/huggingface/diffusers/pull/2545/commits/9e87526c760ed16e8020ee9a3a954286a170e2ae